### ISSUE TITLE
unestimated time label -  functionality

### ISSUE THREAD

**Issue by jrans** [dwyl/tudo#33]
When created an issue unestimated label automatically given by backend?

When estimated time added need time unestimated removed by backend.

**Comment by nelsonic** [comment]
This is _functionality_ we need to _build_. 

> As an organisation/repo owner using **Tudo** to Organise & Prioritise GitHub Issues I would like the _option_ of automatically assigning a Lable of "**Time-Unestimated**" to all new issues/stories which have not yet been estimated. 
> 
> Once an issue has been estimated the "**Time-Unestimated**" label should _automagically_ be removed from the issue.

@iteles how does that sound?

**Comment by iteles** [comment]
I would rephrase slightly to:

> As an organisation/repo owner using **Tudo** to Organise & Prioritise GitHub Issues I would like the _option_ of automatically assigning a **"Time-Unestimated"** _label_ to all new issues/stories which have not yet been estimated.
> Once an the time an issue will take to resolve has been estimated and a **Time** label has been added ([#34](https://github.com/dwyl/tudo/issues/34)) the "Time-Unestimated" label should _automagically_ be removed from the issue.

But :+1:

**Comment by nelsonic** [comment]
I see your rephrase and re-raise you:

> "Arrr! As an organisation/repo owner usin' Tudo t' Organise & Prioritise GitHub Issues I would like t' option o' automatically assignin' a "Time-Unestimated" label t' all new issues/stories which have not yet been estimated. Once t' time an issue will take t' resolve has been estimated and a Time label has been added (#34) t' "Time-Unestimated" label should automagically be removed from t' issue."

Courtesy of http://postlikeapirate.com/ :wink:

**Comment by iteles** [comment]
:laughing:

**Comment by samhstn** [comment]
This is no longer the desired app functionality, closing
